.ci/azure-pipelines.yml

```diff
@@ -42,10 +47,6 @@ jobs:
       vmImage: 'ubuntu-latest'
 
     steps:
-      - script: 'echo "##vso[task.setvariable variable=JELLYFIN_VERSION]$(git describe --tags)"'
-        condition: startsWith(variables['Build.SourceBranch'], 'refs/tags')
-        displayName: 'Set Version Variable'
-
```

Comment:
Well GH does not allow for suggestions on deleted lines :expressionless: 

```yaml
- displayName: 'Set Job Scoped Variable (used by gradle)'
  condition: startsWith(variables['Build.SourceBranch'], 'refs/tags/v')
  script: 'echo "##vso[task.setvariable variable=JELLYFIN_VERSION]$[replace(variables['Build.SourceBranch'], 'refs/tags/', '')]"'
```

NOTE: I am not 100% sure if this will work, but according to the Azure Pipeline docs I am confident it should. 
Reason for me thinking this should work:
* $[\<expression\>] is a runtime expression, that gets processed during runtime (see [here](https://docs.microsoft.com/en-us/azure/devops/pipelines/process/variables?view=azure-devops&tabs=yaml%2Cbatch#understand-variable-syntax))
* ##vso[task.setvariable variable=JELLYFIN_VERSION] is the way already used to set the Variable for the downstream steps (see [here](https://docs.microsoft.com/en-us/azure/devops/pipelines/process/variables?view=azure-devops&tabs=yaml%2Cbatch#set-a-job-scoped-variable-from-a-script))

The only alternative that comes to my mind is duplicating the `Build` job :thinking:

Reply:
probably ignore this, I think @EraYaN is the correct way to solve this :P